.github/workflows/storybook-check.yml

```diff
@@ -25,3 +26,19 @@ jobs:
 
             - name: Build Storybook
               run: npm run storybook:build
+
+            - name: Install Playwright dependencies
+              run: npx playwright install
+
+            - name: Serve Storybook and run tests
+              run: |
+                  npx concurrently -k -s first -n "SB,TEST" -c "magenta,blue" \
+                  "npm run storybook:dev" \
```

Comment:
Can we just serve (`http-server`) the built storybook like in the [example](https://storybook.js.org/docs/writing-tests/test-runner#run-against-non-deployed-storybooks)? That is what I hoped would save time, instead of rebuilding with `storybook:dev`.